modules/nodes-cluster-admission-about.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-admission.adoc
+
+[id="nodes-containers-controllers-about_{context}"]
+= Understanding custom admission controllers
+
+Admission control plug-ins intercept requests to the master API prior to persistence of a resource, but after the request is authenticated and authorized.
+
+Each admission control plug-in is run in sequence before a request is accepted into the cluster. If any plug-in in the sequence rejects the request, the entire request is rejected immediately, and an error is returned to the end-user.
+
+Admission control plug-ins may modify the incoming object in some cases to apply system configured defaults. In addition, admission control plug-ins may modify related resources as part of request processing to do things such as incrementing quota usage.
+
+The {product-title} master has a default list of plug-ins that are enabled by
+default for each type of resource (Kubernetes and {product-title}): 
+
+* DefaultStorageClass
+* DefaultTolerationSeconds
+* LimitRanger 
+* MutatingAdmissionWebhook
+* NamespaceLifecycle
+* OwnerReferencesPermissionEnforcement
+* PersistentVolumeClaimResize
+* Priority
+* ResourceQuota
+* ServiceAccount
+* ValidatingAdmissionWebhook
+
+These are required for the proper functioning of the master. Modifying these lists is not
+recommended. Future versions of the product may use a different set of plug-ins and may change their ordering. If
+you do override the default list of plug-ins in the master configuration file,
+you are responsible for updating it to reflect requirements of newer versions of
+the {product-title} master.
+
+For the full list of admission controllers, see the 
+link:https://v1-13.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-does-each-admission-controller-do[Kubernetes documentation].
+
+AlwaysPullImages::
```

Comment:
I believe this is deprecated. Some of these plugins are deprecated. PTAL - https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-does-each-admission-controller-do

Reply:
@ravisantoshgudimetla I left the deprecated plug-ins in the list if they are in the [Admission Plugin Ownership spreadsheet](https://docs.google.com/spreadsheets/d/1UtdL44W2sLzJCAz8M1eBCTfFMvm9h7-3OT7w9GEzR90/edit#gid=0). I am trying to verify what is in that list.